.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "thirdparty/pybind11"]
```

Comment:
Dependency should be installed outside the project, except some very special code that will be considered an integrated part in the source form, e.g., [metal-cpp](https://github.com/solvcon/modmesh/blob/master/thirdparty/metal-cpp.sh).

One way to build the dependency is to use https://github.com/solvcon/devenv .

Reply:
Moving docker files into `/contrib`

Reply:
Perhaps I didn't make it clear.  What I meant was that there should not be sub-module in the repository and devenv is one way we did to avoid it.

Reply:
I do remove the submodule
Please check the `Files changed` tab for detail.
I'm just telling you I also move all the docker files into `/contrib`.

Reply:
But the file is still in your changeset: https://github.com/dboyliao/modmesh/commit/730bef36c6e76806bd7c9da9e6385c42bea28f89

Reply:
Oh, I forgot to remove the file.
My bad.
Sorry.